content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md

```diff
@@ -0,0 +1,116 @@
+---
+reviewers:
+- sig-cluster-lifecycle
+title: Certificate Management with kubeadm
+content_template: templates/task
+---
+
+{{% capture overview %}}
+
+This page explains various kubeadm-specific topics related to certificate management
```

Comment:
i think this should be under our folder in setup/independent. WDYT?
the idea was to move all kubeadm pages to setup/ eventually, except the reference docs.

Reply:
This was the location @Bradamant3 and I agreed on. `--use-api`, for example, isn't setup-related, it's only ever used after a cluster has been set up and running for a while.